src/main/index.ts

```diff
@@ -7,14 +7,22 @@
 
 import * as Mobx from "mobx";
 import { spawn } from "node-pty";
+import process from "process";
 import * as LensExtensionsCommonApi from "../extensions/common-api";
 import * as LensExtensionsMainApi from "../extensions/main-api";
 import { getDi } from "./getDi";
 import startMainApplicationInjectable from "./start-main-application/start-main-application.injectable";
 
 const di = getDi();
 
-void di.inject(startMainApplicationInjectable);
+(async () => {
+  try {
+    await di.inject(startMainApplicationInjectable);
+  } catch (error) {
+    console.error(error);
```

Comment:
Could be `di.inject(logErrorInjectable)(error);`.

Reply:
Maybe but since this is failing to start I figured that we don't actually know if logger could be available. This is mostly a last ditch effort to get some information back before exiting.

Reply:
Roger.